packages/cadl-ranch-specs/http/parameters/spread/main.tsp

```diff
@@ -185,6 +185,42 @@ namespace Alias {
   @put
   op spreadAsRequestBody(...BodyParameter): NoContentResponse;
 
+  model ModelParameter {
+    name: string;
+  }
+
+  alias BodyParameterAliasWithModel = {
+    @path
+    id: string;
+
+    ...ModelParameter;
+
+    @header
+    `x-ms-test-header`: string;
+  };
```

Comment:
Could you double check with other .NET devs? I think this case is under design change https://github.com/Azure/typespec-azure/issues/772

Reply:
I think that change doesn't impact payload. How about we remove the "should generate a model named XXX" and just test payload?

Reply:
It does not impact payload, but it would impact what this test tests for...

Reply:
Correct.

Reply:
fixed

Reply:
Same here. People have doubts for this. Let them confirm and resolve themselves.